### ISSUE TITLE
CudaTensor with CPU

### ISSUE THREAD

**Issue by HarbingerOfHell** [badripatro/PQG#4]
HI!

I'm using a CPU config for the eval (I'm using the pre-trained model). However, I get this error 

{
  batch_size : 150
  val_images_use : 24800
  txtSize : 512
  optim_epsilon : 1e-08
  att_size : 512
  input_encoding_size : 512
  learning_rate_decay_start : 5
  id : "1"
  emb_size : 512
  optim_beta : 0.999
  rnn_size : 512
  cnn_dim : 512
  language_eval : 1
  learning_rate_decay_every : 5
  optim : "rmsprop"
  gpuid : -1
  drop_prob_lm : 0.5
  input_ques_h5 : "data/quora_data_prepro.h5"
  checkpoint_dir : "Results/checkpoints"
  rnn_layers : 1
  seed : 1234
  input_json : "data/quora_data_prepro.json"
  backend : "nn"
  save : "Results"
  iterPerEpoch : 1250
  start_from : "pretrained/model_epoch7.t7"
  max_iters : -1
  momentum : 0.9
  save_checkpoint_every : 2500

when I run it by changing the gpuid to -1 and backend to nn. 
What could be a solution. The torch.load does not allow for map_location=torch.device('cpu')

**Comment by vinodkkurmi** [comment]
@HarbingerOfHell  Hi this code runs on GPU without changing anything.   For running it to CPU, all the CUDA Tensors should be converted to Tensor. Only by changing the gpuid to -1 , won't work.
